scripts/poddbmismatch.py

```
@@ -0,0 +1,235 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import os
+import shutil
+import subprocess
+import sys
+from typing import Dict, List, Optional, Set
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)